src/nbl/ext/ImGui/shaders/vertex.hlsl

```diff
@@ -1,27 +1,36 @@
 #include "common.hlsl"
+#include "psinput.hlsl"
 
-[[vk::push_constant]] struct PushConstants pc;
+[[vk::push_constant]] struct nbl::ext::imgui::PushConstants pc;
+
+struct VSInput
+{
+    [[vk::location(0)]] float2 position : POSITION;
+    [[vk::location(1)]] float2 uv : TEXCOORD0;
+    [[vk::location(2)]] float4 color : COLOR0;
+};
 
 /*
     we use Indirect Indexed draw call to render whole GUI, note we do a cross 
     platform trick and use base instance index as replacement for gl_DrawID 
     to request per object data with BDA
 */
 
-PSInput VSMain(VSInput input, uint drawID : SV_InstanceID)
+nbl::ext::imgui::PSInput VSMain(VSInput input, uint drawID : SV_InstanceID)
 {
-    PSInput output;
+    nbl::ext::imgui::PSInput output;
```

Comment:
`using namespace` could be handy here too!

Reply:
resolved